pages/basics/content/getting-started.js

```diff
@@ -1,6 +1,30 @@
 module.exports = {
   title: 'Getting Started',
-  subtitle: '',
-  description: ``.replace(/~/g, '`'),
+  subtitle: 'start using Glamorous',
+  description: `
+    <a href="https://youtu.be/lmrQTpJ_3PM" title="glamorous walkthrough">
+    <img src="https://github.com/paypal/glamorous/raw/master/other/glamorous-walkthrough.png" alt="Video Screenshot" title="Video Screenshot" width="700" />
+  </a>
+
+  Here are a basic example that show you how to get started making component with ~css-in-js~ magic
```

Comment:
Shouldn't it be better if this was being written as:
Here is a basic example that shows you how to get started making a component with `css-in-js` magic.